src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -223,10 +249,76 @@ public Command followTrajectory(String name, PathPlannerTrajectory trajectory){
         ).beforeStarting(() -> trajectoryLog.append("Following trajectory " + name)
         ).andThen(() -> trajectoryLog.append("Trajectory " + name +  " Ended"));
     }
-    
+
+    public Command alignToTarget(NodePosition nodePosition){
+        Integer id = Photonvision.getAprilTagID();
+        double xOffset = 0; //Range offset //TODO: Add constants to calculate 
+        double yOffset = 0; //Left/Right offset //TODO: Add TOF sensor method to calculate
```

Comment:
Is this the game element offset?  Is there a different offset for high and medium?  Maybe not

Reply:
Don't know what you mean by game element offset

There will likely be differing offsets, but I don't know yet, 0 is just a placeholder